src/Utilities/PaymentUtils.res

```diff
@@ -1,5 +1,7 @@
+let paymentMethodListValue = Recoil.atom("paymentMethodListValue", PaymentMethodsRecord.defaultList)
```

Comment:
It's causing a cyclic dependency that's why it's been created a new place. There was no other way. And if we don't put any other recoil then unnecessary computation we have to do everywhere.

```.res
let plist = paymentlist->getDictFromJson->PaymentMethodsRecord.itemToObjMapper
```
I have to extract and do this operation instead of 
```res
  // File - PaymentElement.res
  let (list, setList) = React.useState(_ => PaymentMethodsRecord.defaultList)
```

keeping this as a localstate I made it as a Global state.

I find this optimal because I don't have to do this operation everywhere as it can take time. Instead of that we can keep the result at some place. Any other solutions, I'll take up